src/ButtonStyles.js

```diff
@@ -42,25 +42,50 @@ const getButtonStyles = theme => {
         border-color: ${get('colors.button.border')(theme)}; //convert black to rbg here
       }
 
-      &:selected {
-        background-color: ${get('colors.button.activeBg')(theme)};
-        background-image: none;
-        box-shadow: ${get('colors.blackfade15')(theme)} 0px 0.15em 0.3em inset;
-        border-color: ${get('colors.button.border')(theme)};
-      }
-
       &.disabled {
         color: ${get('colors.button.disabledColor')(theme)}!important;
         background-color: ${get('colors.gray.1')(theme)}!important;
         background-image: none!important;
         border-color: ${get('colors.blackfade20')(theme)}!important;
         box-shadow: none!important;
+        cursor: default;
       }
 
       &:focus {
         outline: none;
         box-shadow: ${get('colors.button.focusShadow')(theme)} 0px 0px 0px 0.2em;
       }
+
+      &.grouped {
+        position: relative;
+        border-right-width: 0;
+        border-radius: 0;
+
+        &:first-child {
+          border-top-left-radius:  ${get('radii.1')(theme)}px;
+          border-bottom-left-radius: ${get('radii.1')(theme)}px;
+        }
+
+        &:last-child {
+          border-right-width: 1px;
+          border-top-right-radius: ${get('radii.1')(theme)}px;
+          border-bottom-right-radius: ${get('radii.1')(theme)}px;
+        }
+
+        &:focus,
+        &:active,
+        &:hover {
+          border-right-width: 1px;
+
+          + .grouped {
+            border-left-width: 0;
```

Comment:
This is neat! Adding back the right border on hover makes the button group feel very polished 👍